src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const list = [...document.querySelectorAll('li')];
+const ul = document.querySelector('ul');
+
+function sortList(employeeList) {
+  const salary = employeeList.sort((
+    a, b) => +b.dataset.salary.split('$').join('').split(',').join('')
+    - +a.dataset.salary.split('$').join('').split(',').join(''));
+
+  for (const item of salary) {
+    ul.append(item);
+  }
+
+  return ul;
```

Comment:
```suggestion
  ul.append(...sortedEmployees);
```
Возвращать из функции ничего не надо